### PR TITLE
doc: add minutes for build wg meeting 2016-10-11

### PR DESCRIPTION
**Pull request by mhdawson**
add minutes for build wg meeting 2016-10-11

### CODE REVIEW

File: doc/meetings/build-wg-meetings-2016-10-11.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Previous meeting: https://docs.google.com/document/d/1mz-gvvFlaZaz1Fdl_fM5oLGtOHMRugul4XjkR537UnM/edit
+
+Next meeting: November 1, 2016
+
+
+## Present
+
+* Johan Bergström (@jbergstroem)
+* Michael Dawson (@mhdawson)
+* JoãReis (@joaocgreis)
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meetings-2016-10-11.md]: João Reis

File: doc/meetings/build-wg-meetings-2016-10-11.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+* Rich Trott (@Trott)
+* Michele Capra (listening in)
+* Gibson Fahnestock (@gibfahn)
+* Myles Borins (@TheAlphaNerd) 
+
+## Standup
+
+* JoãReis
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meetings-2016-10-11.md]: João Reis

**Comment by jbergstroem** [comment]
:shipit:

**Comment by mhdawson** [comment]
Landed as https://github.com/nodejs/build/commit/0b648b62b32618f7c326a2f3d8f8937c477ce0b9

**Pull request closed without merging**